patches/minecraft/net/minecraft/world/World.java.patch

```diff
@@ -59,9 +59,11 @@
      }
  
      public boolean func_175667_e(BlockPos p_175667_1_)
-@@ -308,24 +327,50 @@
+@@ -307,25 +326,52 @@
+         }
          else
          {
++            p_180501_1_ = p_180501_1_.func_185334_h(); // Forge - prevent mutable BlockPos leaks
```

Comment:
Move this down a few lines so it just above
`net.minecraftforge.common.util.BlockSnapshot blockSnapshot = null;`
That way it the forge patches are all together in one section.

Reply:
`p_180501_1_` is already used in the next line. This is to guarantee safety.

Reply:
That was the reasoning, yes, but I can move it if desired.

Reply:
The next line is `getChunkFromBlockCoords` which I think is very unlikely to store the pos no matter what the implementation is. 
By default it's just `return this.getChunkFromChunkCoords(pos.getX() >> 4, pos.getZ() >> 4);`. 
Thoughts?